extensions/theme-abyss/package.nls.json

```diff
@@ -1,5 +1,5 @@
 {
 	"displayName": "Abyss Theme",
-	"description": "Abyss theme for Visual Studio Code",
```

Comment:
@smartguest Is there anything special we have to do for changing loc strings in VS Code extensions?